detection-rules/qr_code_suspicious_indicators.yml

```diff
@@ -53,7 +55,35 @@ source: |
           strings.icontains(sender.display_name, .email.domain.sld)
       )
       or length(body.current_thread.text) is null
-      or body.current_thread.text == ""
+      or (
```

Comment:
just to make sure i'm understanding this logic correctly, we're adding a new condition that when there is at least one reference, the suspicious indicator is that it doesn't have "In-Reply-To" header and the subject doesn't indicate a reply/forward, which would normally be included when a message includes references. 

New logic breakdown: 
// current thread is empty (could be evaded by "Sent with Microsoft Outlook" auto signatures, warning banners, or actual signatures)
AND 
  // it has references
  // but doesn't have a "In-Reply-To" header
  // and the subject doesn't indicate a reply/forward
  OR
  // it has no references.

Reply:
Correct, except that `current thread is empty` isn't actually new, it's just the git diff being silly. I was adding extra logic to that check